pkg/daemon/metrics.go

```diff
@@ -55,11 +55,11 @@ var (
 			Help: "state of kubelet health monitor",
 		})
 
-	// MCDRebootErr logs success/failure of reboot and errors
+	// MCDRebootErr logs failure of reboot and tallys count
 	MCDRebootErr = prometheus.NewGaugeVec(
 		prometheus.GaugeOpts{
```

Comment:
Same as `MCDPivotErr` above
```suggestion
	MCDRebootErr = prometheus.NewCounterVec(
		prometheus.CounterOpts{
```